sqlmesh/core/model/meta.py

```diff
@@ -241,7 +244,9 @@ def interval_unit(self, sample_size: int = 10) -> IntervalUnit:
             croniter = CroniterCache(self.cron)
             samples = [croniter.get_next() for _ in range(sample_size)]
             min_interval = min(b - a for a, b in zip(samples, samples[1:]))
-            if min_interval >= 86400:
+            if min_interval >= 2419200:
+                self._interval_unit = IntervalUnit.MONTH
+            elif min_interval >= 86400:
                 self._interval_unit = IntervalUnit.DAY
             elif min_interval >= 3600:
                 self._interval_unit = IntervalUnit.HOUR
```

Comment:
Should we use constants here? E.g. `SECONDS_IN_MONTH`, etc.

Reply:
It isn't used elsewhere, so I'm not sure how much value the constant would provide. I also didn't want to deviate from how the rest of the method worked.

Reply:
It's probably fine either way, just thought it might make it easier to read, compared to decoding that these numbers are the # of seconds in the corresponding unit. 🤷